bikeshare-server.js

```diff
@@ -4,15 +4,6 @@ var app = express();
 app.set('view engine', 'jade');
 app.use(express.static('public'));
 
-// const requestHandler = (request, response) => {  
```

Comment:
it's good that you removed this, but i am just pretend like there is something wrong with it, so "fix it" ;)